react-ui/src/pages/submission/SubmissionForm.jsx

```diff
@@ -172,13 +178,23 @@ function SubmissionForm({
       let errorValue = null;
       switch (field) {
         case 'category':
-          if (!categories.length) {
-            break;
+          if (categories.length && !formState[field].value) {
+            errorValue = 'This is a required question. ';
           }
-        // Fall through
+          break;
         case 'description':
+          if (!formState[field].value) {
+            errorValue = 'This is a required question. ';
+          } else if (formState[field].value.length > CHAR_LIMIT_DESCRIPTION) {
+            errorValue = `Description exceeds max length of ${CHAR_LIMIT_DESCRIPTION}. `;
+          }
+          break;
         case 'name':
-          errorValue = formState[field].value ? null : 'This is a required question';
+          if (!formState[field].value) {
+            errorValue = 'This is a required question.';
```

Comment:
If you choose to keep these together, this is missing a space that is in the error above. Having to remember to include spaces is a bit of a pain compared to using margins.

In any case, my preferences would be to only show the error messages, so in that case, no spaces should be present in any string.

Reply:
See my comment above about line break